pkg/apis/messaging/v1/subscription_validation.go

```diff
@@ -32,6 +35,13 @@ func (s *Subscription) Validate(ctx context.Context) *apis.FieldError {
 		original := apis.GetBaseline(ctx).(*Subscription)
 		errs = errs.Also(s.CheckImmutableFields(ctx, original))
 	}
+	if feature.FromContext(ctx).IsEnabled(feature.CrossNamespaceEventLinks) {
+		flag := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"))
+		crossNamespaceError := cn.CheckNamespace(ctx, s, flag)
```

Comment:
@Cali0707 @pierDipi Do you think it's necessary to pass the flag in `CheckNamespace`? The flag is only used to check if it's enabled/disabled in the function, but I already checked this before calling it.

Reply:
I think if you already check it then there is no need to pass the flag, good catch!